### PR TITLE
Initial commit of the rcx-tk galaxy wrapper

### PR DESCRIPTION
**Pull request by KristinaGomoryova**
Initial commit of the galaxy wrapper for the rcx-tk tool. 
closes https://github.com/RECETOX/galaxytools/issues/552

### CODE REVIEW

File: tools/rcx-tk/rcx-tk.xml [hunk 1]
```diff
@@ -0,0 +1,70 @@
...
+        <person
+                givenName="Kristina"
+                familyName="Gomoryova"
+                url="https://github.com/KristinaGomoryova"
+                identifier="0000-0003-4407-3917" />
+        <person
+                givenName="Helge"
+                familyName="Hecht"
+                url="https://github.com/hechth"
+                identifier="0000-0001-6744-996X" />
```
Comment (hechth) [inline tools/rcx-tk/rcx-tk.xml:29]: Please also add RECETOX as an organization :)

File: tools/rcx-tk/rcx-tk.xml [hunk 2]
```diff
@@ -0,0 +1,70 @@
...
+        #set $input_file_new = 'input.tsv'
+    #else
+        #set $input_file_new = 'input.' + str($input_file.ext)
+    #end if
+       cp ${input_file} ${input_file_new} &&
+       python3 -m rcx_tk --method ${method} ${input_file_new} 'output.tsv' 
+    ]]></command>
+    <inputs>
+        <param name="input_file" type="data" format="csv,tabular,tsv,xls,xlsx,txt" label="Input data"/>
+        <param name="method" type="select" label="Method">
```
Comment (hechth) [inline tools/rcx-tk/rcx-tk.xml:42]: We could add help texts to those parameters.

**Review by hechth — APPROVED** [review]

File: tools/rcx-tk/rcx-tk.xml (hunk 1 above)
Comment (KristinaGomoryova) [reply tools/rcx-tk/rcx-tk.xml:29]: resolved in cab382ef227760397b9ea570fea5583a3c00dd42

File: tools/rcx-tk/rcx-tk.xml (hunk 2 above)
Comment (KristinaGomoryova) [reply tools/rcx-tk/rcx-tk.xml]: resolved in 241bbcd0b26161fb7c4e21f7cb49798e558cb899

**Pull request merged**